### PR TITLE
feat(publisher): expose config as flags

### LINKED ISSUE #3552: deis-publisher should be less verbose -- add log_level setting
It appears that the deis-publisher unit is uselessly filling up my log with lines like these:

```
Apr 22 17:40:50 Deis-2 logger:  2015-04-22T15:40:50UTC deis-publisher[1]: 2015/04/22 15:40:50 updateDir /deis/services/adapter
Apr 22 17:40:51 Deis-2 logger:  2015-04-22T15:40:51UTC deis-publisher[1]: 2015/04/22 15:40:51 set /deis/services/cms/cms_v6.web.1 -> 172.17.8.102:49160
Apr 22 17:40:51 Deis-2 logger:  2015-04-22T15:40:51UTC deis-publisher[1]: 2015/04/22 15:40:51 set /deis/services/adapter/adapter_v16.web.1 -> 172.17.8.103:49156
Apr 22 17:40:51 Deis-2 logger:  2015-04-22T15:40:51UTC deis-publisher[1]: 2015/04/22 15:40:51 updateDir /deis/services/cms
Apr 22 17:40:51 Deis-2 logger:  2015-04-22T15:40:51UTC deis-publisher[1]: 2015/04/22 15:40:51 updateDir /deis/services/adapter
Apr 22 17:40:52 Deis-2 logger:  2015-04-22T15:40:52UTC deis-publisher[1]: 2015/04/22 15:40:52 set /deis/services/auth/auth_v5.web.1 -> 172.17.8.102:49159
Apr 22 17:40:52 Deis-2 logger:  2015-04-22T15:40:52UTC deis-publisher[1]: 2015/04/22 15:40:52 set /deis/services/panel/panel_v10.web.1 -> 172.17.8.103:49154
Apr 22 17:40:52 Deis-2 logger:  2015-04-22T15:40:52UTC deis-publisher[1]: 2015/04/22 15:40:52 updateDir /deis/services/panel
Apr 22 17:40:52 Deis-2 logger:  2015-04-22T15:40:52UTC deis-publisher[1]: 2015/04/22 15:40:52 updateDir /deis/services/auth
```

Apparently the publisher is constantly rewriting etcd keys for my apps even if they are already there.

### PR DESCRIPTION
**Pull request by eMxyzptlk**
This PR implements the first point as per [3577#issuecomment-97982351](https://github.com/deis/deis/pull/3577#issuecomment-97982351)

closes #3552

### CODE REVIEW

File: publisher/main.go [hunk 1]
```diff
@@ -12,34 +13,42 @@ import (
...
+	dockerHost      = flag.String("docker-host", defaultDockerHost, "The host where to find docker.")
+	etcdHost        = flag.String("etcd-host", defaultEtcdHost, "The etcd host.")
+	etcdPort        = flag.String("etcd-port", defaultEtcdPort, "The etcd port.")
+	logLevel        = flag.String("log-level", defaultLogLevel, "Acceptable values: error, debug")
+)
 
 func main() {
-	endpoint := getopt("DOCKER_HOST", "unix:///var/run/docker.sock")
-	etcdHost := getopt("ETCD_HOST", "127.0.0.1")
+	runtime.GOMAXPROCS(runtime.NumCPU())
```
Comment (bacongobbler) [inline publisher/main.go]: could you please pull this out in another PR?

Comment (eMxyzptlk) [inline publisher/main.go]: Oh sorry, I missed that from my stash.

Comment (eMxyzptlk) [inline publisher/main.go]: done

**Comment by bacongobbler** [comment]
code LGTM

**Comment by mboersma** [comment]
Code LGTM.

**Comment by carmstrong** [comment]
Code LGTM.
